api/src/modules/api-events/api-event.api.entity.ts

```diff
@@ -32,6 +32,9 @@ export enum API_EVENT_KINDS {
   scenario__costSurface__shapeConversionFailed__v1_alpha1 = 'scenario.costSurface.shapeConversionFailed/v1alpha1',
   scenario__costSurface__costUpdateFailed__v1_alpha1 = 'scenario.costSurface.costUpdateFailed/v1alpha1',
   scenario__costSurface__finished__v1_alpha1 = 'scenario.costSurface.finished/v1alpha1',
+  project__wpda__submitted__v1__alpha = 'project.wdpa.submitted/v1/alpha',
+  project__wpda__submitted__v1__finished = 'project.wdpa.finished/v1/alpha',
+  project__wpda__submitted__v1__failed = 'project.wdpa.failed/v1/alpha',
```

Comment:
please double check the labels of these enum members - did you mean `project__wdpa__finished__v1__alpha` and so on? (or `project__protectedAreas__finished__v1__alpha` if replacing wdpa)

Reply:
🤦 shame on me! thanks for catching this!